tests/Operation/Factory/SubresourceOperationFactoryTest.php

```diff
@@ -75,7 +75,7 @@ public function testCreate()
                 ],
                 'route_name' => 'api_dummy_entities_subresource_get_subresource',
                 'path' => '/dummy_entities/{id}/subresource.{_format}',
-                'operation_name' => 'subresource_get_subresource',
+                'operation_name' => 'api_dummy_entities_subresource_get_subresource',
```

Comment:
isn't this a BC break?

Reply:
Well, IMO yes and no. Yes because, even if generated, it could have been used. But no because I think it's a bug and that it should always have been like that. And no because subresources are experimental and that part is internal.

Reply:
It should at least be documented in the changelog as a BC break.